### PR TITLE
Add support RTL88x2BU usb wifi dongles

### PR DESCRIPTION
**Pull request by kyangk**
This pull request adds support for RTL88x2BU usb wifi dongles.

Package source is from `https://github.com/cilynx/rtl88x2bu`

I have personally tested RTL8812BU on my rpi3b+ running Homeassistant OS 32bit w/ `Kangzio USB WiFi Adapter 1300Mbps`.

```
# lsmod
Module                  Size  Used by
88x2bu               2338816  0
rfcomm                 49152  4
nf_conntrack_netlink    49152  0
nfnetlink              16384  2 nf_conntrack_netlink
...
```

```
# ./iwconfig
...
wlan1     IEEE 802.11AC  ESSID:"REDACTED"  Nickname:"<WIFI@REALTEK>"
          Mode:Managed  Frequency:5.785 GHz  Access Point: REDACTED
          Bit Rate:867 Mb/s   Sensitivity:0/0
          Retry:off   RTS thr:off   Fragment thr:off
          Encryption key:****-****-****-****-****-****-****-****   Security mode:open
          Power Management:off
          Link Quality=77/100  Signal level=70/100  Noise level=0/100
          Rx invalid nwid:0  Rx invalid crypt:0  Rx invalid frag:0
          Tx excessive retries:0  Invalid misc:0   Missed beacon:0
```

Some other example hardware (from searching web)
```
Fastoe AC1200 USB Wi-Fi Adapter
Cudy WU1200 AC1200 High Gain USB Wi-Fi Adapter
TP-Link Archer T3U
TP-Link Archer T3U Plus
TP-Link Archer T4U V3
Linksys WUSB6400M
Dlink DWA-181
Dlink DWA-182
```

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @kyangk, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;operating-system#1742).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: buildroot-external/package/rtl88x2bu/Config.in [hunk 1]
```diff
@@ -0,0 +1,6 @@
+config BR2_PACKAGE_RTL88X2BU
+	tristate "Realtek 88x2 USB WiFi"
+	depends on BR2_LINUX_KERNEL
+	---help---
+	  Help message of RTL88X2BU
```
Comment (agners) [inline buildroot-external/package/rtl88x2bu/Config.in:5]:
```suggestion
	  Driver for RTL88X2BU USB WiFi adapters (no firmware needed).
```

**Review by agners — COMMENTED** [review]
Can you also add the driver to generic-x86-64, khadas-vim3, and the odroids?

**Comment by kyangk** [comment]
All the other added platforms compiled successfully. I have no way of actually testing them beyond this.

**Comment by kyangk** [comment]
Anything else I should do for this to merge?

File: buildroot-external/package/rtl88x2bu/Config.in [hunk 2]
```diff
@@ -0,0 +1,6 @@
+config BR2_PACKAGE_RTL88X2BU
+	tristate "Realtek 88x2 USB WiFi"
+	depends on BR2_LINUX_KERNEL
+	---help---
+	  Driver for RTL88X2BU USB WiFi adapters (no firmware needed).
+
```
Comment (agners) [inline buildroot-external/package/rtl88x2bu/Config.in:6]:
```suggestion
```

**Review by agners — CHANGES_REQUESTED** [review]

**Comment by diegomontanaduarte** [comment]
@agners When this merge will be available on rel-7 branch?

**Comment by agners** [comment]
Currently I don't intend to backport it to rel-7. rel-8 will soon enter rc phase. Also, you can install nightly builds already (https://os-builds.home-assistant.io/), it will be possible to upgrade to rc/stable builds.

**Comment by diegomontanaduarte** [comment]
@agners any ETA for rel-8 stable release?

**Comment by agners** [comment]
I'll build a first rc next week or so, final release depends on how it goes... I hope in April still. We will see.

**Comment by diegomontanaduarte** [comment]
@agners Niceeeeee! Thanks for your prompt response on all my questions!

**Pull request merged**